WKSMissionUnit.yml

```diff
@@ -9,13 +9,15 @@ fields:
         targets: [ClassJobCategory]
   - name: Unknown3
   - name: Unknown4
-  - name: Unknown5
-  - name: Unknown6
+  - name: SilverStarRequirement
+  - name: GoldStarRequirement
   - name: Unknown7
   - name: Unknown8
   - name: Unknown9
   - name: Unknown10
-  - name: Unknown11
+  - name: WKSMissionToDo
+    type: link
+    targets: [WKSMissionToDo]
```

Comment:
This is not `WKSMissionToDo` but instead `WKSMissionSupplyItem` 
![image](https://github.com/user-attachments/assets/4db4edd1-b80d-40ab-b4d8-b0cdbc853a66)